tests/sample.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * SLA bridge and channel creation tests.
+ *
+ */
+"use strict";
+
+/*global describe:false*/
+/*global afterEach:false*/
+
+var util = require('util');
+var Emitter = require('events').EventEmitter;
+
+var mockClient;
+var mockChannel;
```

Comment:
I don't think you need this since you are storing mock channels for a given test in the channels array.